lib/src/tasks/analyze/cli.dart

```diff
@@ -62,9 +66,9 @@ class AnalyzeCli extends TaskCli {
         fatalWarnings: fatalWarnings,
         hints: hints,
         fatalHints: fatalHints,
+        fatalLints: fatalLints,
         strong: strong);
     var title = task.analyzerCommand;
-    if (fatalHints) title += ' (treating hints as fatal)';
```

Comment:
this was a bit of detritus from when we added our own fatal hints support, figured I would clean it up here